Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:18-alpine AS builder
+
+WORKDIR /app
+
+COPY package*.json /app/
+
+RUN npm ci
+
+COPY . .
+
+ENV NODE_BUILD=true
+
+RUN npm run build
+
+EXPOSE 3000
+
+CMD node build
+
+FROM node:18-alpine as production
```

Comment:
Is there a reason to split the build and run into separate containers?

Reply:
Ok seems fine then, but does there need to be an EXPOSE and CMD for the first stage? Isn't it needed for only the second container?

Reply:
Yeah, @kareemmahlees just remove the `EXPOSE` and `CMD node build` from the build stage.